.github/workflows/dependencies_validator.yaml

```diff
@@ -0,0 +1,31 @@
+name: Wazuh QA Framework Validator
+on:
+  pull_request:
+    paths:
+      - requirements.txt
+      - deps/wazuh_testing/setup.py
+jobs:
+  validate-framework:
+    runs-on: ubuntu-20.04
+    continue-on-error: true
+    strategy:
+      matrix:
+        python-version:
+          - 3.10.9
+          - 3.7.1
```

Comment:
The [PR](https://github.com/wazuh/wazuh-jenkins/pull/5342/files) shows that you are using Python 3.11. However, for this PR, you are using Python 3.7 and Python 3.10. Could you clarify the cause, please?